apps/test/unit/templates/projects/ProjectAppTypeAreaTest.js

```diff
@@ -86,14 +89,17 @@ describe('ProjectAppTypeArea', () => {
           isDetailView={true}
         />
       );
-      expect(wrapper.find('ProjectCard')).toHaveLength(12);
-      expect(wrapper.find('Button').first().text()).toBe('View more');
-      expect(stubAjax).not.toHaveBeenCalled();
+      const projectCards = screen.getAllByText(/Published Project \d+/);
```

Comment:
Do you know if these cards have a `role` of [listitem](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listitem_role)? We can use `screen.getByRole('listitem')` instead and focus it down on the project gallery using RTL's `within` feature.

If the cards do not have that role, does it make sense to add the role? To me, the cards do seem like list items from an accessibility perspective.

Reply:
I don't think the cards have a role:
```
<div class="card">
    <a
        class="link"
        href="/projects/applab/STUB_CHANNEL_ID_1_"
        target="_blank"
    >
        <div class="thumbnail">
            <img
                alt="Project thumbnail image."
                class="image"
                src="/blockly/media/projects/project_default.png"
            />
        </div>
        <div class="title ui-project-name-applab">
            Published Project 1.
        </div>
        <div class="" />
        <div class="lastEdit">
            Published : 
            <time  datetime="2017-01-01T11:00:00.001Z">
                8 years ago
            </time>
        </div>
    </a>
</div>
```

Reply:
I will add `role=list` and `role=listitem`.
I have already tested locally, just waiting for github issue to resolve.